### PR TITLE
 UXPROD-3903: Update schemas to support new bursar export feature

### PR DESCRIPTION
**Pull request by nhanaa**
[UXPROD-3903](https://issues.folio.org/browse/UXPROD-3903): This PR will merge team @folio-org/bama's changes in folio-export-common to support the new bursar export configuration feature. This PR must be merged before `mod-data-export-spring`, `mod-data-export-worker`, and `ui-plugin-bursar-export`.

### CODE REVIEW

**Review by ncovercash — APPROVED** [review]

**Comment by khandramai** [comment]
@nhanaa , could you please squash commits just to not clutter up the master.

**Comment by ncovercash** [comment]
> @nhanaa , could you please squash commits just to not clutter up the master.

@khandramai it can be squashed on merge

**Review by siarhei-charniak — APPROVED** [review]

File: schemas/bursar-export/legacy/legacyBursarFeeFinesTypeMapping.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
       "itemType": {
         "type": "string",
         "minLength": 1,
-        "maxLength": 12
+        "maxLength": 1
```
Comment (khandramai) [inline schemas/bursar-export/legacy/legacyBursarFeeFinesTypeMapping.json:14]: Is this correct? Item type should be the only one symbol?

Comment (nhanaa) [reply schemas/bursar-export/legacy/legacyBursarFeeFinesTypeMapping.json:14]: I think that is a mistake. I will correct this.

**Pull request merged**